(ref) rewriteFrames mention in source maps docs (#1155)
getsentry/sentry-docs: src/collections/_documentation/platforms/javascript/sourcemaps.md

If you’ve uploaded source maps and they aren’t applying to your code in an issue in Sentry, take a look at the JSON of the event and look for the `abs_path` to see exactly where we’re attempting to resolve the file  - i.e. `http://localhost:8000/scripts/script.js` (`abs_path` will appear once for each frame in the stack trace - match this up with the file(s) that are not deminified.). A link to the JSON view can be found at the top of the issue page next to the date the event occurred. The uploaded artifact names must match these values.

{% capture __alert_content-%}

> You may want to use the [`rewriteFrames`]({%- link _documentation/platforms/javascript/index.md -%}#rewriteframes) integration to change your `abs_path` values in the case that you have dynamic values in your path i.e. `https://www.site.com/{some_value}/scripts/script.js`

Reviewer: ```suggestion
If you have dynamic values in your path (for example, `https://www.site.com/{some_value}/scripts/script.js`), you may want to use the [`rewriteFrames`]({%- link _documentation/platforms/javascript/index.md -%}#rewriteframes) integration to change your `abs_path` values.
```
